### ISSUE TITLE
hyperparameter tunning with AI Platform Training: issue with parameter dependency

### ISSUE THREAD

**Issue by tarrade** [tarrade/proj_NLP_text_classification_with_GCP#37]
not possible to put in the .yaml file:
min_df<max_df

```
    params:
    - parameterName: WE_max_df
      type: DOUBLE
      minValue: 1.0
      maxValue: 1.0
      scaleType: UNIT_LINEAR_SCALE
    - parameterName: WE_min_df
      type: DOUBLE
      minValue: 0
      maxValue: 0.3
      scaleType: UNIT_LINEAR_SCALE
```

**Comment by tarrade** [comment]
send to GCP support:
https://console.cloud.google.com/support/cases/detail/21567856?cloudshell=false&organizationId=716917234918&orgonly=true

**Comment by tarrade** [comment]
No good answer for GCP support.

Implemented returning an accuray of 0.0 when the parameters are incorrect for scikit-lear:
![image](https://user-images.githubusercontent.com/12021701/71085080-f5bf7e00-2196-11ea-9bc9-727e17941724.png)

No answer on StackOverflow
https://stackoverflow.com/questions/59343183/how-to-force-parameter-dependency-when-using-ai-platform-hyper-parameter-tuning
